### ISSUE TITLE
Cannot find rule related to docblock changes

### ISSUE THREAD

**Issue by darthf1** [rectorphp/rector#2888]
## Question
If I run rector, with whichever set i choose, changes are made to docblocks which i cannot relate to any of the rules in the set beeing applied.

Example: `vendor/bin/rector process src --dry-run --set=solid`

```diff
    ---------- begin diff ----------
--- Original
+++ New
@@ -16,12 +16,12 @@
 use Doctrine\Common\Persistence\ManagerRegistry;

 /**
- * @method MyEntity|null find($id, $lockMode = null, $lockVersion = null)
- * @method MyEntity|null findOneBy(array $criteria, array $orderBy = null)
+ * @method (MyEntity| null) find($id, $lockMode = null, $lockVersion = null)
+ * @method (MyEntity| null) findOneBy(array $criteria, array $orderBy = null)
  * @method MyEntity[]    findAll()
  * @method MyEntity[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
  */
-class MyEntityRepository extends ServiceEntityRepository
+final class MyEntityRepository extends ServiceEntityRepository
 {
     public function __construct(ManagerRegistry $registry)
     {
    ----------- end diff -----------

Applied rules:

 * Rector\SOLID\Rector\Class_\FinalizeClassesWithoutChildrenRector
```
Question: Which rule is responsible for this change? (the docblock one, not the final one)

Another one:

```diff
---------- begin diff ----------
--- Original
+++ New
@@ -88,14 +88,10 @@
  * @ORM\Table(
  *     name="my_table_name",
  *     indexes={
- *         @ORM\Index(
- *             name="my_table_name_idx", columns={"x", "xx", "xxx", "xxxx"}
- *         ),
- *         @ORM\Index(
- *             name="my_table_name_xxx_idx", columns={"xxx"}
+ *         @ORM\Index(name="my_table_name_idx",
+ *             columns={"x", "xx", "xxx", "xxxx"}), @ORM\Index(name="my_table_name_xxx_idx",
+ *         columns={"xxx"})}
  *         )
- *     }
- * )
  * @ORM\Entity(repositoryClass="App\Repository\MyEntityRepository")
  */
 class MyEntity
```
Question: Which rule is responsible for this change?

**Comment by TomasVotruba** [comment]
It's not related to any Rector rule, but to the part responsible for printing docblocks. It's not perfect, as there is no docblock parser, so there are some conventions for spacing and text positions (e.g. space after comma `,`).

- The first issue should be fixed in  `master`. Could you verify?
- The second looks like space-preserving printer for docblocks isuses. I'll investigate. Could you provide demo link with minimal code: https://getrector.org/demo

**Comment by darthf1** [comment]
Thanks!

- First one is indeed fixed in master!
- Demo: https://getrector.org/demo/37429caa-1af3-413a-aea1-41dbda709f49#result

**Comment by darthf1** [comment]
Awesome. Thanks.

**Comment by darthf1** [comment]
@TomasVotruba I noticed that the second issue is fixed in the 0.7.1 release, but that first one is not. Was it not part of the release?

**Comment by TomasVotruba** [comment]
It was in the master, so it was.

Could add failing test case with PR to phpdoc info test to confirm?

Also try to remove vendor and composer.lock. it might be buggy sometimes
